docs/layers/heatmap.md

```diff
@@ -0,0 +1,62 @@
+# HeatMaps
+
+This directory contains documentation on adding heatmap layers.\
+For full documentation follow this [link](https://docs.mapbox.com/help/tutorials/make-a-heatmap-with-mapbox-gl-js/).
+
+## Sample heatmap layer
+
```

Comment:
Maybe add a single line here saying that the values below are the default values? Not sure if that's true for `heatmap-weight`?